buildSrc/src/main/groovy/DependencyResolver.groovy

```diff
@@ -1,3 +1,17 @@
+/* Copyright 2016 Google LLC
```

Comment:
Fix year and formatting of license (or is that year correct?)

Reply:
This is actually correct - the file was created in 2016, I am backdating the license.

This format matches what we use in the rest of gapic-generator.